tempesta_fw/http_msg.c

```diff
@@ -811,3 +815,105 @@ tfw_http_msg_alloc(int type)
 
 	return hm;
 }
+
+/**
+ * Remove Hop-by-Hop headers in terms of RFC 7230 6.1
+ */
+int tfw_http_rm_hbh_hdrs(TfwHttpMsg *hm)
```

Comment:
As noted by @ikoveshnikov, these functions should be named `tfw_http_msg_*_hbh_hdrs()` as they reside in `http_msg.c` module.